cfgmgr/nbrmgr.cpp

```diff
@@ -64,6 +65,20 @@ NbrMgr::NbrMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, con
                               TableConsumable::DEFAULT_POP_BATCH_SIZE, default_orch_pri);
     auto consumer = new Consumer(consumerStateTable, this, APP_NEIGH_RESOLVE_TABLE_NAME);
     Orch::addExecutor(consumer);
+          
+    string swtype;
+    Table cfgDeviceMetaDataTable(cfgDb, CFG_DEVICE_METADATA_TABLE_NAME);
+    if(cfgDeviceMetaDataTable.hget("localhost", "switch_type", swtype))
+    {
+        //If this is voq system, let the neighbor manager subscribe to state of SYSTEM_NEIGH
+        //entries. This is used to program static neigh and static route in kernel for remote neighbors.
+        if(swtype == "voq")
+        {
+            string tableName = STATE_SYSTEM_NEIGH_TABLE_NAME;
+            Orch::addExecutor(new Consumer(new SubscriberStateTable(stateDb, tableName, TableConsumable::DEFAULT_POP_BATCH_SIZE, 0), this, tableName));
```

Comment:
why we are not calling Orch::addConsumer() here ? We don;t have to repeat this code

Reply:
Yes. We can call Orch::addConsumer() here. I'll fix this.